### PR TITLE
Copy and Paste error for Include Processor

### PR DESCRIPTION
**Pull request by ecki**
Fixed a title and unified capsing

### CODE REVIEW

File: docs/asciidoctorj.adoc [hunk 1]
```diff
@@ -1352,7 +1352,7 @@ public class UriIncludeProcessor extends IncludeProcessor { // <1>
 <3> `push_include` method inserts new content (retrieved from the url) in current position of document.
 
 [source]
-.Register an Inline macro processor
+.Register an Include processor
```
Comment (mojavelinux) [inline docs/asciidoctorj.adoc]: Can you lowercase include here and down below? We are speaking about "include" in a general sense here, so it shouldn't be a proper noun.

Comment (ecki) [inline docs/asciidoctorj.adoc]: Yes if you think thats correct. All other processors are written uppercase, however.

**Comment by ecki** [comment]
Capsing looks like

	Line 865: |Include processor
	Line 1294: === Include processor
	Line 1299: .Include processor example
	Line 1355: .Register an include processor
	Line 1370: .Example of include processor

now

**Comment by mojavelinux** [comment]
Most excellent. Thanks!

**Pull request merged**